app/lib/pdf_filler/nj1040_pdf.rb

```diff
@@ -57,6 +57,92 @@ def hash_for_pdf
         "undefined_9": get_line_7_exemption_count,
         "x  1000_2": get_line_7_exemption_count * 1000,
       }
+
+      dependents = get_dependents
+      if dependents[0]
```

Comment:
What are your thoughts on packaging this up into a function with a map from {dependent indices -> [various pdf field indices]}? I don't think a change is necessary, as the code here feels much more explicit & readable, and maybe the duplication is fine given that this code shouldn't change very much.